### ISSUE TITLE
class .ui-inputwrapper-filled is not removed on p-autocomplete with multipe=true

### ISSUE THREAD

**Issue by makbari** [primefaces/primeng#4753]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a bug report
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
floating label is not get back to its position on clearing the autocomplete.
Also, floating labels do not work with initial value on input.

**Expected behavior**
Floating label should get back to it's initial position.

`<span class="ui-float-label">`
                    `<p-autoComplete id="trainer" [(ngModel)]="trainer" [suggestions]="filteredTrainer" (onClear)="changeModel()"
                                    (onSelect)="changeModel()" (onUnselect)="changeModel()"
                                    (completeMethod)="filterTrainer($event)" [minLength]="1"
                                    [multiple]="true"
                                    [dropdown]="true">`
                    `</p-autoComplete>`
                    `<label for="trainer">Trainer ({{filteredTrainer?.length}})</label>`
                  `</span>`

* **Angular version:** 5.0.X

* **PrimeNG version:** 5.0.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by rafa-as** [comment]
Same here!
